scripts/deploy-to-gcs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#set -x
+
+set -eu
+
+# Publish the helm charts to GCS.  This is primarily intended to be run by Travis but is
+# usable standalone (assuming you have docker installed and are running from the helm-charts directory).
+
+# Use caution with the -d flag.  If you happen to run this against the production repo but from a local
+# helm-charts dir with several tarballs missing for some reason, those tarballs will be removed from the
+# production repo.  Deletion is disabled by default.
+
+# This implementation inspired by https://medium.com/google-cloud/google-cloud-sdk-dockerfile-861a0399bbbb
+
+usage () {
+    echo "${0##*/} [-d] [-n] | [-h]"
+    echo "  -d   Delete objects in repo that are not in docs dir.  Default is false."
+    echo "  -n   Dry run.  Show what would be published.  Default is false"
+    echo "  -h   Print this help."
+    exit 1
+}
+
+# GCS project
+GCS_PROJECT=es-repo
+# GCS bucket within project
+: ${GCS_BUCKET:=marcoderama-test}
```

Comment:
Is this the final name...
Can we choose bucket names like production and stating 
es-producution
es-staging

Another thing that confuses me the right to use the word "es",  Almost all commercial offerings from LB are under enterprise-suite.

Reply:
That's a temporary bucket name.  I don't want to use an 'official' one until all the other pieces are in place.  e.g. HTTPS redirect.  I don't know if we'll have staging and production versions.  (In the mean time, we'll publish to this private bucket.  Seemed okay to me.)

The project name was chosen by Karl.  Are you suggesting that should change, or just the bucket name?

Reply:
Can we set this to something a bit more generic like `helm-charts-test`. Then in `.travis.yml` we can override it with `helm-charts` as the bucket name.

Reply:
Unfortunately bucket names need to be globally unique and `helm-charts-test` is already taken.  `lb-helm-charts-test` and `lightbend-helm-charts-test` are available.  I'll go with the latter unless someone suggests otherwise.

(For that matter, I guess it's not entirely 'correct' that these live in the `es-repo` project, since the charts are not necessarily `es`-related.  Not sure if I have the power to create a new repo or not.  Looks like it from the UI.  That doesn't seem to get much (any?) external exposure though so I'd guess we can probably live with `es-repo`.  $0.02)